.github/workflows/autoLabelDuplicate.yml

```diff
@@ -9,6 +9,6 @@ jobs:
     steps:
       - uses: Amwam/issue-comment-action@v1.3.1
         with:
-          keywords: '["duplicate of"]'
+          keywords: '["duplicate of #", "duplicate of https://github.com/FreeTubeApp/FreeTube/issues/"]'
```

Comment:
```suggestion
          keywords: '["duplicate of #", "duplicate of https://github.com/FreeTubeApp/FreeTube/issues/", "duplicate of https://github.com/FreeTubeApp/FreeTube/pulls/"]'
```

Reply:
ah okay, srry about that potatohead.exe wasnt responding